kyo-prelude/shared/src/main/scala/kyo2/Abort.scala

```diff
@@ -17,11 +17,12 @@ object Abort:
     given eliminateAbort: Reducible.Eliminable[Abort[Nothing]] with {}
     private inline def erasedTag[E]: Tag[Abort[E]] = Tag[Abort[Any]].asInstanceOf[Tag[Abort[E]]]
 
-    inline def fail[E](inline value: E): Nothing < Abort[E] =
-        Effect.suspendMap[Any](erasedTag[E], Fail(value))(_ => ???)
+    inline def fail[E](inline value: E): Nothing < Abort[E] = error(Fail(value))
 
-    inline def panic[E](ex: Throwable): Nothing < Abort[E] =
-        Effect.suspendMap[Any](erasedTag[E], Panic(ex))(_ => ???)
+    inline def panic[E](ex: Throwable): Nothing < Abort[E] = error(Panic(ex))
+
+    inline def error[E](e: Error[E]): Nothing < Abort[E] =
```

Comment:
Shouldn't `e` be inlined here?

Do we want to make this package private? It should be stable, but I prefer to keep the public API small.

Reply:
good catch! I've added the missing inline in `panic` as well

Reply:
> Do we want to make this package private? It should be stable, but I prefer to keep the public API small.

Makes sense, I've made it `private[kyo]`.